cmd/ipfs_routingd/.go-ipfs/datastore/LOG

```diff
@@ -0,0 +1,9 @@
+=============== Feb 6, 2015 (MST) ===============
```

Comment:
ok it looks like the `.go-ipfs` dir was added by mistake (all these leveldb files)